include/tvm/relay/expr.h

```diff
@@ -221,6 +221,13 @@ class FunctionNode : public BaseFuncNode {
    */
   bool IsPrimitive() const;
 
+  /*!
+   * \brief Check whether the function is marked as inline.
+   *
+   * \return Whether the function should be inlined or not.
+   */
+  bool IsInline() const;
```

Comment:
Is `IsMarkedInline` a better name? IsInline seems like the function is already inlined.